otelgorm/README.md

```diff
@@ -44,6 +44,14 @@ if err := db.WithContext(ctx).Raw("SELECT 42").Scan(&num).Error; err != nil {
 
 See [example](/example/) for details.
 
+## Usage: exceptions
+
+In case you want to omit certain database operations from any trace/span, you can also set:
+```go
+db.Set(EnabledTraceDatabaseKey, false)
```

Comment:
Shouldn't that be `otelgorm.EnabledTraceDatabase`?

Reply:
Changed as you asked, and also made the readme slightly more descriptive.